DRACOON (OAuth).cyberduckprofile

```diff
@@ -0,0 +1,573 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Help</key>
+        <string>https://help.cyberduck.io/dracoon</string>
+        <key>Protocol</key>
+        <string>dracoon</string>
+        <key>Vendor</key>
+        <string>dracoon-oauth</string>
+        <key>Description</key>
+        <string>DRACOON (OAuth)</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Username Placeholder</key>
+        <string>Username</string>
+        <key>Password Placeholder</key>
+        <string>Password</string>
+        <key>Default Hostname</key>
+        <string>dracoon.team</string>
+        <key>Hostname Configurable</key>
+        <true/>
+        <key>Port Configurable</key>
+        <true/>
+        <key>Anonymous Configurable</key>
+        <false/>
+        <key>Path Configurable</key>
+        <false/>
+        <key>Certificate Configurable</key>
+        <false/>
+        <key>Authorization</key>
+        <string>oauth</string>
+        <key>Username Configurable</key>
+        <false/>
+        <key>Password Configurable</key>
+        <false/>
+        <key>OAuth Authorization Url</key>
+        <string>/oauth/authorize</string>
+        <key>OAuth Token Url</key>
+        <string>/oauth/token</string>
+        <key>OAuth Redirect Url</key>
+        <string>/oauth/callback</string>
+        <key>OAuth Client ID</key>
+        <string>dracoon_desktop</string>
+        <key>OAuth Client Secret</key>
+        <string>DEmTMv59zUlqvRv6zHbVCKYylEy8Ds9Y</string>
+        <key>Scopes</key>
```

Comment:
Necessary? Don't have it in Cyberduck nor in their own build.

Reply:
Providing explicit scope was required at some point otherwise a `400` was returned.

Reply:
Should be consistent with the ones in their own build.